doc/docs/security/selinux.md

```diff
@@ -0,0 +1,44 @@
+<!-- markdownlint-disable-file MD013-->
+# BlueChi's SELinux policy
+
+BlueChi provides a custom SELinux policy, limiting access of the `bluechi-controller` and `bluechi-agent`. It can be installed via
+
+```bash
+dnf install bluechi-selinux
+```
+
+By default, the package allows `bluechi-controller` to bind and `bluechi-agent` to connect to any port so that changes of the [ControllerPort](../man/bluechi-agent-conf.md#controllerport-uint16_t) are not being blocked.
+
+## Enforce port restrictions
+
+In order to allow BlueChi to only use one port, the usage of any needs to be disabled with [setsebool](https://linux.die.net/man/8/setsebool):
+
+```bash
+# Turn off the any policy for bluechi-controller
+setsebool -P bluechi_controller_port_bind_any 0
+
+# Turn off the any policy for bluechi-agent
+setsebool -P bluechi_agent_port_connect_any 0
+```
+
+Subsequently, the desired port needs to be allowed by using [semanage](https://linux.die.net/man/8/semanage):
+
+```bash
+# Set the allowed port for bluechi-controller
+semanage port -a -t bluechi_port_t -p tcp <port>
+
+# Set the allowed port for bluechi-agent
+semanage port -a -t bluechi_agent_port_t -p tcp <port>
+```
+
+## Change from enforcing to permissive
```

Comment:
I think we are trying to avoid permissive mode in general, so is it worth to have that section?

Reply:
I think its not bad to have it. Its not much effort and if somebody for some reason really wants to switch it to permissive, it briefly described here.